utils/defaults.stanza

```diff
@@ -62,7 +63,18 @@ public pcb-board default-board (num-layers:Int, outline:Shape) :
 
 ; Creates a design using the default board. 
 public defn make-default-board (module:Instantiable, num-layers:Int, outline:Shape) :
-  set-main-module(module)
+  
+  var intermediate-design = transform-module(propagate-rail-voltage, module)
```

Comment:
this looks suspicious to me - how do we know all the rail voltages have been propagated by the end?

Reply:
We don't. The pass running needs more work, but this works for power systems that are less than 3 regulators deep...